src/cmd/run.go

```diff
@@ -57,6 +69,41 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := "opslevel-runner-leader-lock"
+	leaseLockNamespace := viper.GetString("pod-namespace")
+	podName := viper.GetString("pod-name")
+
+	config, err := pkg.GetKubernetesConfig()
+
+	if err != nil {
+		log.Info().Msgf("Failed to get kubeconfig")
+	}
+
+	client := k8sClientset{
```

Comment:
Did you make this type just so you could add the `getNewLock` function?  Lets undo that - you can just make a utility function where you pass in the client as the first argument.